Formula/e/erlang.rb

```diff
@@ -73,6 +87,8 @@ def install
     system "./configure", *std_configure_args, *args
     system "make"
     system "make", "install"
+    (buildpath/"bin").install resource("ex_doc") => "ex_doc"
```

Comment:
Oops, this snippet doesn't work. Let's do this:
```suggestion
    resource("ex_doc").stage do |r|
      (buildpath/"bin").install File.basename(r.url) => "ex_doc"
    end
```